src/Data/CurrentUser/errorToast.tsx

```diff
@@ -3,6 +3,7 @@ import { toast } from 'react-toastify'
 import { ClientError } from 'scrivito'
 
 export function errorToast(title: string, error: unknown): void {
+  console.log('errorToast', title, error)
```

Comment:
* [ ] `console.error` is the better choice b/c it provides the developer with the appropriate UI
* [ ] FYI an abstraction for logging errors could be nice at some point to allow for easy integration of Honeybadger etc.